pkg/scheduler/core/extender.go

```diff
@@ -114,6 +115,36 @@ func NewHTTPExtender(config *schedulerapi.ExtenderConfig) (algorithm.SchedulerEx
 	}, nil
 }
 
+// Equal is used to check if two extenders are equal
+// ignoring the client field, exported for testing
+func Equal(extender1, extender2 *HTTPExtender) bool {
```

Comment:
Is there a cleaner way to do this comparison? Like, looping over the fields and checking them (with the possibility to skip certain fields)?

Reply:
probably with `reflect.Value`, but I think this is good enough.
Prefer shorter variable names, like `e1`